db/migrate/20240219143649_change_document_tags.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class ChangeDocumentTags < ActiveRecord::Migration[7.1]
+  def change
+    add_column :documents, :string_tags, :string, array: true, default: []
```

Comment:
Might want to look at adding a GIN index to speed up queries based on tags:
https://www.postgresql.org/docs/current/indexes-types.html#INDEXES-TYPES-GIN